frontend/src/components/homepage/PreviewModal.css

```diff
@@ -0,0 +1,64 @@
+.preview-modal-header-box {
```

Comment:
Can you move these to their own files in `frontend/src/theme/components/` and import them in `frontend/src/theme/index.ts`? 
See the chakra ui for examples https://github.com/uwblueprint/planet-read/pull/49/files

Reply:
I got rid of the file entirely and moved all styling to each <Box>